app/views/home/how_government_works.html.erb

```diff
@@ -1,6 +1,5 @@
 <% page_title "How government works" %>
 <% page_class "inside_gov_how-gov-works" %>
-<% meta_description "In the UK, the Prime Minister leads the government with the support of the Cabinet and ministers. You can find out who runs government and how government is run, as well as learning about the history of government." %>
```

Comment:
Hmm, the commit message is a little misleading, as it does appear that the meta description is [making it through to the rendered page](https://www.gov.uk/government/how-government-works) (via [application_helper](https://github.com/alphagov/whitehall/blob/master/app/helpers/application_helper.rb#L12-L14) and [frontend base](https://github.com/alphagov/whitehall/blob/cf70ac9f379ca638cb735b83e0ea02f585ee5d4b/app/views/layouts/_frontend_base.html.erb#L11)):

```html
<meta name="description" content="In the UK, the Prime Minister leads the government with the support of the Cabinet and ministers. You can find out who runs government and how government is run, as well as learning about the history of government.">
```

However, it makes sense to use a consistent approach throughout Whitehall.

Reply:
Oops, I've amended the commit. (I actually thought those pages weren't being used anymore, as well! Thanks for correcting me there - wouldn't want to delete how government works.)